Orange/data/io.py

```diff
@@ -29,6 +29,9 @@
 from Orange.util import Registry, flatten, namegen
 
 
+__all__ = ["Flags"]
```

Comment:
I don't think that's ok. The module defines a lot more "public" stuff than just `Flags`. Perhaps rather make `import *` in _Orange.data.\_\_init\_\_.py_ explicit?